cameraview/src/main/java/com/otaliastudios/cameraview/engine/Camera2Engine.java

```diff
@@ -1403,6 +1411,26 @@ protected boolean applyPreviewFrameRate(@NonNull CaptureRequest.Builder builder,
         return false;
     }
 
+    private void sortRanges(Range<Integer>[] fpsRanges) {
+        if (mPreviewFrameRateExact) {
```

Comment:
Please see my Camera1Engine comment. It would make sense to always sort in descending order when the desired fps is unset (== 0), to have a flexible default.